mappings/net/minecraft/network/packet/s2c/play/TeamS2CPacket.mapping

```diff
@@ -7,6 +7,22 @@ CLASS net/minecraft/class_5900 net/minecraft/network/packet/s2c/play/TeamS2CPack
 	FIELD field_29152 teamName Ljava/lang/String;
 	FIELD field_29153 playerNames Ljava/util/Collection;
 	FIELD field_29154 team Ljava/util/Optional;
+	FIELD field_33346 ADD I
+		COMMENT The {@link #packetType} that creates a new team with a few players.
+	FIELD field_33347 REMOVE I
+		COMMENT The {@link #packetType} that removes a team.
+	FIELD field_33348 UPDATE I
+		COMMENT The {@link #packetType} that updates a team's information.
+	FIELD field_33349 ADD_PLAYERS I
+		COMMENT The {@link #packetType} that adds a few players to a team.
+	FIELD field_33350 REMOVE_PLAYERS I
+		COMMENT The {@link #packetType} that removes a few players from a team.
+	FIELD field_33351 FIRST_NAME_OR_VISIBILITY_RULE_LENGTH I
+		COMMENT One of the name tag visibility rule or collision rule strings' max length.
+		COMMENT Has value {@value}.
+	FIELD field_33352 SECOND_NAME_OR_VISIBILITY_RULE_LENGTH I
+		COMMENT One of the name tag visibility rule or collision rule strings' max length.
+		COMMENT Has value {@value}.
```

Comment:
IMO we should keep them unmapped if we aren't sure what they are. Javadoc looks fine.

Reply:
Imo these mapping is still more informative than having it unmapped. Also currently due to some problems, javadoc can only be added on mapped elements. Both fields are `0`.

Reply:
Oh, okay then. 👍